server/routes/templates.ts

```diff
@@ -27,6 +27,10 @@ import {
 import * as Branches from "../branch";
 import * as crypto from "crypto";
 
+import {allowInsecurePrototypeAccess} from "@handlebars/allow-prototype-access";
```

Comment:
Do we actually use object prototypes anywhere in our Handlebars code in order to require this?

Reply:
Yes, for instance, all the user objects on the frontend.  It was very broken without this, after we updated past Handlebars 4.6.0